### PR TITLE
Adding pymove package

### PR DESCRIPTION
**Pull request by flych3r**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/pymove/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    - pymove.preprocessing
+    - pymove.uncertainty
+    - pymove.utils
+    - pymove.visualization
+
+about:
+  home: "https://github.com/InsightLab/PyMove"
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
```
Comment (synapticarbors) [inline recipes/pymove/meta.yaml:73]: Add a comment above the `license_file` line that includes a link to the following PR (https://github.com/InsightLab/PyMove/pull/66), which would, if merged, allow you to remove the manually packaged copy of the license file and use one that comes in the source distribution. The comment is meant as a reminder to check for this change in future updates to the feedstock.

File: recipes/pymove/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: b6d8dfbe9a6b7f0cb8587a70db5bfc385c0a9395a5e430a61f38ec45198e5b77
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - branca
```
Comment (synapticarbors) [inline recipes/pymove/meta.yaml:18]: under `host`, I believe you only need `python` and `pip`. Everything else is a `run` dependency only.

File: recipes/pymove/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: b6d8dfbe9a6b7f0cb8587a70db5bfc385c0a9395a5e430a61f38ec45198e5b77
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```
Comment (synapticarbors) [inline recipes/pymove/meta.yaml:14]: I believe this package can be made `noarch: python`. See:

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

In that case, change `python` to `python >=3.5` under `host` and `run`

**Comment by flych3r** [comment]
> I'm still trying to work out what the package version conflict is. Unfortunately it's not always obvious from the logs.

Thanj you, the builds passed with the proposed changes to `build` and `host`

**Comment by synapticarbors** [comment]
I think it may have been that I updated a dependency, `holidays`, which feedstock was out of date, but I'm not sure.

**Comment by synapticarbors** [comment]
@flych3r -- thank you for your contribution to conda-forge. If you run into any other issues once the feedstock is created, please ping us.

**Pull request merged**